source/_docs/configuration/remote.markdown

```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
 
 </div>
 
+## Setting up remote access
 <div class='note'>
-Home Assistant no longer support remote access via IP address since release 0.77, you have to use a domain name.
+Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name.  A Domain can be obtained cheaply from several providers. 
 </div>
 
-The most common approach is to set up port forwarding (for any port) from your router to port 8123 on the computer that is hosting Home Assistant. General instructions on how to do this can be found by searching `<router model> port forwarding instructions`. You can use any free port on your router and forward that to port 8123.
+To set up a remote instance, you will need to set up Network Address Translation, Port Forwarding on your router, to port 8123 on the computer that is hosting Home Assistant. 
```

Comment:
I get that these is the correct technical terms, however, a regular end-user has no clue what network address translation is.

Let's just keep it at port forwarding, there is no need to overcomplicate things for those people.

Reply:
removed "Network Address Translation"